client_restaurant/shared/src/commonMain/kotlin/data/gateway/LocalConfigurationGateway.kt

```diff
@@ -0,0 +1,52 @@
+package data.gateway
+
+import data.local.model.UserConfigurationCollection
+import domain.gateway.ILocalConfigurationGateway
+import io.realm.kotlin.Realm
+import io.realm.kotlin.ext.query
+
+class LocalConfigurationGateway(private val realm: Realm) : ILocalConfigurationGateway {
+
+    override suspend fun saveAccessToken(token: String) {
+        realm.write {
+            copyToRealm(UserConfigurationCollection().apply {
+                this.accessToken = token
+            })
+        }
+    }
+
+    override suspend fun getAccessToken(): String {
+        return realm.query<UserConfigurationCollection>().first().find()?.accessToken
+            ?: throw Exception("Access Token not found")
+    }
+
+    override suspend fun saveRefreshToken(token: String) {
+        realm.write {
+            copyToRealm(UserConfigurationCollection().apply {
+                this.refreshToken = token
+            })
+        }
```

Comment:
The same here, the saved record by **_saveAccessToken_** will be completely different from this record saved by **_saveRefreshToken_**.
Fix this please.

Reply:
Thanks so much for your review, we already doing the fix for this class, but just for now we pushed the independent old code for others to can work on their refactoring.